README.md

```diff
@@ -47,7 +47,9 @@ Sending an email message:
 #### Parameters
 
 * `to`: _(mandatory)_ A list of recipients as strings.
+* `subject_text`: _(optional)_ Plain text email subject, instead of `subject`.
```

Comment:
Could you make clear that either the `subject_text` *or* the `subject` option have to be supplied. The same applies for the `body` and the `body_text`.